doc/release-notes/release-8.sgml.in

```diff
@@ -157,6 +157,15 @@ This section gives an account of those changes in three categories:
 	<p>Removed <em>LM_Group</em> helper. The LM protocol is
 	   insecure and no longer supported on the market since 2008.
 
+	<tag>--disable-arch-native</tag>
+	<p>The <em>-march=native</em> compiler option is not used by
+		default. It is possible to enable it by using the
+		<em>--enable-arch-native</em> option instead.
+		This optimization may cause problems when Squid is
+		run on a system with a different exact CPU model than the one
+		it is built on, or in some containerized environments.
+		The symptom is crashes with "illegal instruction" errors.
+		Use with caution.
```

Comment:
I recommend removing this advice as too vague:

```suggestion
```

or making it more specific


```suggestion
		We do not recommend enabling this optimization in virtualized environments.
```

It would be good to have some specific ammunition to reject "illegal instruction" bug reports for `-march=native` builds...

I do not insist on this change.